lib/instance-validator.js

```diff
@@ -112,7 +112,7 @@ class InstanceValidator {
           .then(newError => { throw newError || error; }))
       )
       .then(() => runHooks('afterValidate', this.modelInstance, this.options))
-      .return(this.modelInstance);
+      .return();
```

Comment:
what I mean is, yes, `validate` should never fulfill with the model, but you are not changing `validate` here, but a different method that may be called in other places. It would be great if you could verify the return value is not passed to the user in other places as well.